### PR TITLE
Refactor video and update other references

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Refactor video and update other references

[stage-2]

## Motivation and Context
Component Navigation Improvements

## How Has This Been Tested?
Tested locally.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: src/scripts/template-editor/directives/dtv-component.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-    function (templateEditorFactory, basicStorageSelectorFactory) {
+  .directive('templateComponent', ['templateEditorFactory', 'storageManagerFactory',
+    function (templateEditorFactory, storageManagerFactory) {
       return {
         restrict: 'E',
         templateUrl: 'partials/template-editor/component.html',
         link: function ($scope) {
           $scope.factory = templateEditorFactory;
-          $scope.basicStorageSelectorFactory = basicStorageSelectorFactory;
+          $scope.storageManagerFactory = storageManagerFactory;
```
Comment (ezequielc) [inline src/scripts/template-editor/directives/dtv-component.js:11]: @alex-deaconu Does this need to be added to the $scope here, given you inject it to components? Thanks!

**Review by ezequielc — APPROVED** [review]

File: src/scripts/template-editor/directives/dtv-component.js (hunk 1 above)
Comment (alex-deaconu) [reply src/scripts/template-editor/directives/dtv-component.js:11]: @ezequielc yes, because of the list/thumbnail toggle in `component.html`. Thanks.

Comment (ezequielc) [reply src/scripts/template-editor/directives/dtv-component.js:11]: Got it. Thanks!

**Pull request merged**